content/docs/1.3.4/deploy/install/_index.md

```diff
@@ -61,10 +61,12 @@ curl -sSfL https://raw.githubusercontent.com/longhorn/longhorn/v{{< current-vers
 Example result:
 
 ```shell
-[INFO]  Required dependencies are installed.
+[INFO]  Required dependencies 'kubectl jq mktemp' are installed.
+[INFO]  Hostname uniqueness check is passed.
 [INFO]  Waiting for longhorn-environment-check pods to become ready (0/3)...
 [INFO]  All longhorn-environment-check pods are ready (3/3).
 [INFO]  Required packages are installed.
+[INFO]  MountPropagation is enabled.
```

Comment:
```suggestion
[INFO] Mount propagation is enabled.
```

Reply:
I assume that you are referring to the feature and not a specific field or function in the code. If my assumption is correct, we should not use CamelCase here.